reasoning/src/commonTest/kotlin/mocks/TestModel.kt

```diff
@@ -0,0 +1,56 @@
+package mocks
+
+import com.xebia.functional.tokenizer.ModelType
+import com.xebia.functional.xef.llm.Chat
+import com.xebia.functional.xef.llm.Embeddings
+import com.xebia.functional.xef.llm.models.chat.*
+import com.xebia.functional.xef.llm.models.embeddings.EmbeddingRequest
+import com.xebia.functional.xef.llm.models.embeddings.EmbeddingResult
+import com.xebia.functional.xef.llm.models.usage.Usage
+import kotlinx.coroutines.flow.Flow
+
+class TestModel(override val modelType: ModelType, override val name: String) :
```

Comment:
This is fine, but this testing component can be used in other modules and should be in core in the tests there and consumed here as a library. /cc @javipacheco

Reply:
How should I put the core dependency in reasoning to import that TestModel I tried this but doesn't work

```kotlin
val commonTest by getting {
            dependencies {
                implementation(libs.kotest.property)
                implementation(libs.kotest.framework)
                implementation(libs.kotest.assertions)
                implementation(libs.mock.engine.test)
                implementation(projects.xefCore)
            }
        }
```